client/fraud-scripts/forter.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
+
+export default ( { site_id } ) => {
+	// IDs in HTML can't start with a number, so calling querySelector('script#123456') breaks.
+	if ( ! document.querySelector( 'script[id="' + site_id + '"]' ) ) {
+		const script = document.createElement( 'script' );
+		// Add the ID in case the Forter script looks for it in the page.
+		script.id = site_id;
+		// TODO: This is sent to the checkout JS bundle too. That's not ideal, it's pretty big.
+		// esline-disable-next-line
```

Comment:
Should this be eslint? The lint is failing with: `11:1   error  This line has a length of 5512. Maximum allowed is 140`

Reply:
🤦  Yes, of course